lib/virtual-fs/src/host_fs.rs

```diff
@@ -33,37 +35,98 @@ impl Default for FileSystem {
     fn default() -> Self {
         Self {
             handle: Handle::current(),
+            root: "/".into(),
```

Comment:
Should we keep a default? It seems highly insecure to allow `/` access by default